garage/tf/algos/npo.py

```diff
@@ -33,14 +33,33 @@ class NPO(BatchPolopt):
     """
     Natural Policy Gradient Optimization.
 
-    Attributes:
-        name(str): The name of the algorithm.
+    Args:
+        pg_loss(enum): The type of loss functions to use.
         lr_clip_range(float): The limit on the likelihood ratio between
             policies, as in PPO.
         max_kl_step(float): The maximum KL divergence between old and new
             policies, as in TRPO.
+        optimizer(object): The optimizer of the algorithm.
+        optimizer_args(dict): The arguments of the optimizer.
+        name(str): The name of the algorithm.
         policy_ent_coeff(float): The coefficient of the policy entropy.
-        optimizer(float): The optimizer of the algorithm.
+        entropy_method(str): A string from: 'max', 'regularized'. The type
+            of entropy method to use. 'max' adds the dense entropy to the
+            reward for each time step. 'regularized' adds the mean entropy
+            to the surrogate objective. See https://arxiv.org/abs/1805.00909
+            for more details.
+            !!NOTE!!: make sure when setting entropy_method to 'max', set
```

Comment:
Please put this as a note outside the argument list.

```python
"""
Args:
    foo (str): Stuff

Notes
    Notes about some stuff
"""
```

See example here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html